### PR TITLE
Express setup

### LINKED ISSUE #35: remove Node.js server code
Remove the Node.js server code to make room for the Express code

### PR DESCRIPTION
**Pull request by dyazdani**
## Description
Completed steps for setting up Express, React, and TypeScript. Unresolved issue with seeding DB related to node not being able to find file path to module. Here is a screenshot of the issue:
Screen Shot 2023-09-21 at 21 58 41

## Related Issue
Closes #35 and #36 

## Checklist N/A
- [ ] Created tests which fail without the change (if possible) N/A
- [ ] Confirmed all tests are passing N/A

### CODE REVIEW

**Review by paloobi — CHANGES_REQUESTED** [review]
Remove node_modules and re-submit

File: client/index.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { createRoot } from 'react-dom/client';
+
+import App from './App';
+
+const root = createRoot(document.getElementById("app")!)
```
Comment (paloobi) [inline client/index.tsx:6]: Extra ! was added here, typo I'm guessing?

```suggestion
const root = createRoot(document.getElementById("app"))
```

File: server/index.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import express, { Request, Response, NextFunction } from "express";
+import path from "path";
+
+const app = express();
+
+app.use(express.static(path.join(__dirname, "../public")));
+
+app.get("/", (req: Request, res: Response, next: NextFunction): void => {
+  try {
+    res.send("index.html");
```
Comment (paloobi) [inline server/index.ts:10]: Just checking: Are you intending this response? It is currently responding with the string "index.html" not the actual HTML stored in the index.html

**Review by paloobi — APPROVED** [review]

File: client/index.tsx (hunk 1 above)
Comment (dyazdani) [reply client/index.tsx:6]: That was actually an intentional bit of syntax for a typescript problem:
Screen Shot 2023-09-27 at 09 48 23

File: server/index.ts (hunk 2 above)
Comment (dyazdani) [reply server/index.ts:10]: This was code that was setup up before TS and React were being used. Now the "/" says "Typescript is awesome" from the React App.tsx component in the client directory. I can delete this if it's unnecessary. It does seem like a string here, although in a pre-TS version, it returned the index.html file we set up.

**Pull request merged**